docs/host/maptiler-cloud.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+category: host
+title: Ready to use API
+description: Serve raster and vector tiles with MapTiler Cloud
+order: 1
+---
+## MapTiler Cloud
+
+We are providing a **free reliable maps API** with maps based on **OpenMapTiles schema** with **OpenStreetMap data**. The whole planet is **updated weekly** with **vector** and **raster** tiles available for everyone. 
```

Comment:
Consider 

MapTiler a.g. is providing


Maybe adding logo somewher?

Reply:
Thanks - fix in https://github.com/openmaptiles/www.openmaptiles.org/commit/bee7ff1f41ac243219cc4275990ea17d332cc418.